.NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDatePeriodExtractorConfiguration.cs

```diff
@@ -13,185 +13,185 @@ public class FrenchDatePeriodExtractorConfiguration : BaseDateTimeOptionsConfigu
+            RegexCache.Get(DateTimeDefinitions.MonthOfRegex, RegexFlags);
 
         // TODO: add regexs below
         public static readonly Regex RangeUnitRegex =
-            new Regex(DateTimeDefinitions.RangeUnitRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.RangeUnitRegex, RegexFlags);
 
         public static readonly Regex InConnectorRegex =
-            new Regex(DateTimeDefinitions.InConnectorRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.InConnectorRegex, RegexFlags);
 
         public static readonly Regex WithinNextPrefixRegex =
-            new Regex(DateTimeDefinitions.WithinNextPrefixRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.WithinNextPrefixRegex, RegexFlags);
 
         public static readonly Regex RestOfDateRegex =
-            new Regex(DateTimeDefinitions.RestOfDateRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.RestOfDateRegex, RegexFlags);
 
         public static readonly Regex LaterEarlyPeriodRegex =
-            new Regex(DateTimeDefinitions.LaterEarlyPeriodRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.LaterEarlyPeriodRegex, RegexFlags);
 
         public static readonly Regex WeekWithWeekDayRangeRegex =
-            new Regex(DateTimeDefinitions.WeekWithWeekDayRangeRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.WeekWithWeekDayRangeRegex, RegexFlags);
 
         public static readonly Regex YearPlusNumberRegex =
-           new Regex(DateTimeDefinitions.YearPlusNumberRegex, RegexFlags);
+           RegexCache.Get(DateTimeDefinitions.YearPlusNumberRegex, RegexFlags);
 
         public static readonly Regex DecadeWithCenturyRegex =
-            new Regex(DateTimeDefinitions.DecadeWithCenturyRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.DecadeWithCenturyRegex, RegexFlags);
 
         public static readonly Regex YearPeriodRegex =
-            new Regex(DateTimeDefinitions.YearPeriodRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.YearPeriodRegex, RegexFlags);
 
         public static readonly Regex ComplexDatePeriodRegex =
-            new Regex(DateTimeDefinitions.ComplexDatePeriodRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.ComplexDatePeriodRegex, RegexFlags);
 
         public static readonly Regex RelativeDecadeRegex =
-            new Regex(DateTimeDefinitions.RelativeDecadeRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.RelativeDecadeRegex, RegexFlags);
 
         public static readonly Regex ReferenceDatePeriodRegex =
-            new Regex(DateTimeDefinitions.ReferenceDatePeriodRegex, RegexFlags);
+            RegexCache.Get(DateTimeDefinitions.ReferenceDatePeriodRegex, RegexFlags);
 
         public static readonly Regex AgoRegex =
-                    new Regex(DateTimeDefinitions.AgoRegex, RegexFlags);
+                    RegexCache.Get(DateTimeDefinitions.AgoRegex, RegexFlags);
```

Comment:
Nitpick: As we're going thorugh this, could you fix the indentation here for consistency?